src/Chapter2.hs

```diff
@@ -620,7 +626,13 @@ Write a function that takes elements of a list only on even positions.
 >>> takeEven [2, 1, 3, 5, 4]
 [2,3,4]
 -}
-takeEven = error "takeEven: Not implemented!"
+takeEven :: [a] -> [a]
+takeEven [] = []
+takeEven (x:xs) = takeEvenHelper True (x:xs)
+  where
+    takeEvenHelper :: Bool -> [a] -> [a]
+    takeEvenHelper _ [] = []
+    takeEvenHelper isEven (y:ys) = if isEven then y : takeEvenHelper False ys else takeEvenHelper True ys
```

Comment:
Very creative solution with a helper function! 🎨 

Alternatively, you could use pattern matching in `takeEven` instead. As you need to take every second element, it is possible to represent with a few pattern matching clauses 🙂